cmd/validator/validator.go

```diff
@@ -160,24 +167,20 @@ func getFlags() (validatorConfig, error) {
 		}
 	}
 
-	groupByCleanString := cleanString("groupby")
-	groupByUserInput := strings.Split(groupByCleanString, ",")
-	groupByAllowedValues := []string{"filetype", "directory", "pass-fail"}
-	seenValues := make(map[string]bool)
+	if *excludeFileTypesPtr != "" && *fileTypesPtr != "" {
```

Comment:
Unless I missed something. I would do that to avoid panics

```suggestion
	if excludeFileTypesPtr != nil && fileTypesPtr != nil && *excludeFileTypesPtr != "" && *fileTypesPtr != "" {
```